### ISSUE TITLE
Balancing

### ISSUE THREAD

**Issue by Senderoth** [Malorolam/LootBags#117]
Less of an issue and more of a suggestion, default values for loot bags need balancing. Regularly pull level 3-4 enchanted diamond gear from common bags, while rarer bags will hand you a medium level pickaxe from a different mod, or some obsidian, etc... Makes it so you need to go balance it yourself, which most people wouldn't want to do.

(Only tested it in latest 1.12.2 build, but I assume it's like that in most versions.)

**Comment by Malorolam** [comment]
I'll take a look at where the loot is sitting in the bags and try to incorporate better pool handling so rarer items are properly rare.  I had things sitting pretty nicely in 1.10, but 1.12 they added a ton of enchanted stuff to the vanilla tables and that messed everything up.

**Comment by glindsey** [comment]
I agree that this needs doing, but I can understand it's also a _really_ difficult problem, especially when other mods are adding stuff to loot tables in ways that don't really make sense. Our server gets really strange loot bag results -- for example, any bag (common through legendary) may contain certain Actually Additions crystals, but then other AA crystals don't show up in any bag at all. Or, another really weird example: two of the vanilla Minecraft records can appear in lootbags, but the others can never appear.

**Comment by Malorolam** [comment]
Well what's causing the issue is the pool system, since now there's weights that are based off of the pools, but the pools are in fixed amounts.
With the pool system, you'd get 1-3 "common" items, 0-2 "uncommon" items, and 0-1 "rare" items, where as before you just got 3-5 items and it was all one big pool.  Which is nice for a loot chest perspective, it lets you have things more segmented and guarantee certain items, but when I'm pulling items I'm losing that pool information so I'm stuck with the weights, which results in a lot of common items with low weights and rare items with higher weights.  I'm working on a way to try and pull the number of items given with a pool and adjusting the weight with that.

**Comment by glindsey** [comment]
Ugh. I see what you mean.

Well, in any case, thanks for creating an absolutely awesome mod, as well as being super-responsive to the issues list! You're head and shoulders above some of the other mod authors out there.

**Comment by TheRealDadbeard** [comment]
I have the same issue. What would be the best way to fix that somehow a little bit?

**Comment by glindsey** [comment]
Manually adjusting the whitelist and blacklist settings in the config helped a ton. There’s still the issue where there’s not a huge difference between the different “levels”, but I think more config tweaking will help with that. One thing to note: if you use JEI and are on a server, you have to copy the same config file to both config and server, or JEI will show the wrong info.

Excellent handle, btw. I am also a dadbeard. 👍
